pom.xml

```diff
@@ -1328,6 +1328,51 @@
             </build>
         </profile>
 
+        <!-- Elasticsearch 6.3+ test environment (default) -->
```

Comment:
This should not be advertised as the default profile, since it is not the default profile.

Reply:
Ah yes sorry, I made is so and then we discussed and changed that but forgot to update the comment.